x-pack/test/functional/apps/lens/group1/smokescreen.ts

```diff
@@ -20,10 +20,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const config = getService('config');
 
   describe('lens smokescreen tests', () => {
+    before(async () => {
```

Comment:
Can you add an `after` task to restore it back when done with the `await timePicker.resetDefaultAbsoluteRangeViaUiSettings();`?

Alternatively this could be applied at `index.ts` level and apply for all suites within that group - I think it is already like that for a couple of groups (see group1 for an example )